docs/data/routes/docs/techniques/azure-deployment/en.md

```diff
@@ -17,8 +17,8 @@ Starting from Sitecore 9.1 release JSS server components can be provisioned alon
 ## Step 2: Install the JSS server components (optional)
 
 In case you already have Sitecore instance in Azure you have to use ARM templates methodology.
-1. Download official released Sitecore JSS WDP package https://dev.sitecore.net/Downloads/Sitecore_JavaScript_Services/110/Sitecore_JavaScript_Services_1100.aspx
-1. Download ARM templates for the appropriate topology available on the GitHub https://github.com/Sitecore/Sitecore-Azure-Quickstart-Templates/tree/master/JSS
+1. Download official released Sitecore JSS WDP package <https://dev.sitecore.net/Downloads/Sitecore\_JavaScript\_Services/110/Sitecore\_JavaScript\_Services\_1100.aspx>
+1. Download ARM templates for the appropriate topology available on the GitHub <https://github.com/Sitecore/Sitecore-Azure-Quickstart-Templates/tree/master/JSS%2011.0.1>
```

Comment:
Thanks for calling attention to these links. Ideally, I'm thinking they shouldn't even be tied to a specific version. How about something like:
```suggestion
1. Download [official released Sitecore WDP packages](https://dev.sitecore.net/Downloads/Sitecore_JavaScript_Services.aspx) for the appropriate JSS version and topology.
1. Download [ARM templates available on GitHub](https://github.com/Sitecore/Sitecore-Azure-Quickstart-Templates) for the appropriate JSS version and topology.
```

Reply:
thank you, corrected as suggested